src/main/assembly/dist/bag/metadata/files/2017/09/files.xsd

```diff
@@ -0,0 +1,133 @@
+    <xs:element name="files">
+        <xs:complexType>
+            <xs:all>
+                <xs:element name="file" type="f:dcterms-elements" maxOccurs="unbounded">
+                    <xs:annotation>
+                        <xs:documentation xml:lang="en">Container for file-specific information with qualified dcterms elements.</xs:documentation>
+                    </xs:annotation>
+                </xs:element>
+            </xs:all>
+            <xs:assert test="
+                    every $fi in f:file
+                        satisfies $fi/dcterms:format"/>
+        </xs:complexType>
+    </xs:element>
+
+    <xs:complexType name="dcterms-elements">
+        <xs:sequence>
+            <xs:group ref="dcterms:elementsAndRefinementsGroup"/>
+        </xs:sequence>
+        <xs:attribute name="filepath" type="xs:string"/>
+    </xs:complexType>
+
+    <xs:element name="accessibleToRights" substitutionGroup="dcterms:accessRights" type="f:EasyFileAccessRightsType">
+        <xs:annotation>
+            <xs:documentation xml:lang="en">
+                Restriction on dcterms:accessRights to indicate the accessRights to the file content.
+                If ommitted, it is implicitly derived from the accessRights in dataset.xml
+                The accessRights to the file-metadata can be specified using visibleToRights.
+
+                Element value MUST conform to EasyFileAccessRightsType.
+
+                See also: http://purl.org/dc/terms/accessRights
+            </xs:documentation>
+        </xs:annotation>
+    </xs:element>
+
+    <xs:element name="visibleToRights" substitutionGroup="dcterms:accessRights" type="f:EasyFileAccessRightsType">
+        <xs:annotation>
+            <xs:documentation xml:lang="en">
+                Restriction on dcterms:accessRights to indicate the accessRights to the file metadata.
+                The accessRights to the file-content can be specified using accessibleToRights.
+                If ommitted, it is implicitly derived from the accessRights in dataset.xml (and will be set to 'ANONYMOUS')
+
+                Element value MUST conform to EasyFileAccessRightsType.
+
+                See also: http://purl.org/dc/terms/accessRights
+            </xs:documentation>
+        </xs:annotation>
+    </xs:element>
+
+    <xs:complexType name="EasyFileAccessRightsType">
+        <xs:annotation>
+            <xs:documentation xml:lang="en">
+                Use on dcterms:accessRights or members of its substitutionGroup.
+                dcterms:accessRights is interpreted as accessibleToRights
+            </xs:documentation>
+        </xs:annotation>
+        <xs:simpleContent>
+            <xs:restriction base="dc:SimpleLiteral">
+                <xs:simpleType>
+                    <xs:restriction base="f:EasyFileAccessCategoryType"/>
+                </xs:simpleType>
+                <xs:attribute ref="xml:lang" use="prohibited"/>
```

Comment:
Why is this needed

Reply:
I think because @lindareijnhoudt doesn't want to accept an `xml:lang` there?

Reply:
I forgot that this was a copy, sorry

Reply:
No, this is in fact new, but @lindareijnhoudt wrote it